HIP/hip-583.md

```diff
@@ -2,221 +2,266 @@
+![lazy-create-with-address.png](../assets/hip-583/lazy-create-with-address.png)
 
-When `CryptoCreate.key` is an `ECDSA` key, the network node may calculate the `public-address` and automatically set the accounts alias value. 
+When a user transfers value (hbars or tokens), they may address their destination account using the standard
+account ID format (`shard.realm.num`), or by using an alias format (`shard.realm.alias`). If the alias is an
+`evm address alias`, then the consensus node will first check to see if such an account exists. If it does,
+then it will use the normal rules for transferring the hbars or tokens (including checking `receiverSigRequired`,
+and all other rules). If the account *does not* exist, then the account will be auto-created with the same rules
+specified in `HIP-32`, except that no key will be present on the newly created account. An account created in this
+manner will be known as a `Hollow Account` and will only be able to receive HBAR and applicable token transfers, but
+cannot take part in transactions submitted by others requiring its signature.
+
+Since Hedera accounts require keys to ensure security, an `ECDSA` key matching the derived `evm address alias` must be
+provided in a future signed transaction issued by the owner of the account to take control of the account. Due to the
+nature of `ECDSA secp256k1` keys, the public key can be extracted from the signature and the signed bytes. It can then
+go through the derivation function to generate the associated `evm address`, which can be compared with the `alias` to
+prove the match and account ownership. This process will complete the creation of the account, taking it from a
+`Hollow Account` to a normal complete account with a valid public key.
+
+Note, in this case the final value of the account alias will remain as the `evm address alias` and not the `key alias`.
+
+`EthereumTransaction` transactions where value is transferred  (i.e `tx.to` and `tx.value` are set) should also support
+this case. This will support users of `Ethereum-native wallets` and eth dev tools who are accustomed to creating and
+crediting an account's balance in one step. In this case th `tx.to` value of the RLP encoded transaction would dictate
+the `evm address alias` to be used.
+
+### Setting `alias` with `CryptoUpdate`
+
+`HIP-32` supports updating the `alias` of an account after the account has been created. This HIP extends that
+support to include `evm address alias`es. As with `CryptoCreate`, ownership of the private key of the `ECDSA` public
+key associated with the `evm address` **must** be asserted on the `CryptoUpdate` by signing the transaction with
+that private key. The key used does **not** have to be in state on any account. As with `HIP-32`, there can only
+be a single `alias` per account (except for the `long-zero alias` which all accounts have), and it is immutable, and
+therefore an account that has an `alias` cannot have it updated. 
```

Comment:
Technically, an account can have up to three aliases linked to it (even before HIP-583):
1. the implicit `long-zero alias` which all accounts have
2. an `ECDSA` public key alias set via auto account creation
3. the public `EVM address` derived from that key

Only in some cases is the public `EVM address` alias actually stored in the `alias` field of the account in state, but it is important to note that this alias belongs to the account and the nodes are aware of it.

Reply:
I agree, this is said very well. I have added language to the HIP which I hope is as good as what you have here. You are exactly right.